### PR TITLE
update suma dependency and add more tests validations

### PR DESCRIPTION
**Pull request by jhomarolo**
I updated the suma dependency and also added some new scenario validations to the validation test file.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,10 +32,15 @@
...
   "dependencies": {
-    "suma": "^1.0.5"
+    "suma": "^1.0.6"
   },
   "devDependencies": {
-    "mocha": "^7.1.1",
-    "nyc": "^15.0.1"
+    "mocha": "^7.2.0",
+    "nyc": "^15.1.0",
+    "eslint": "^6.8.0",
```
Comment (dalssoft) [inline package.json]: I'm still not sure about how eslint is reinforced. does it depends on vs code?

File: sample.js [hunk 2]
```diff
@@ -1,31 +1,29 @@
 var user = entity('User', {
+  username: field(String, {
```
Comment (dalssoft) [inline sample.js]: I think this file is not valid any longer... could you delete it?

**Review by dalssoft — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (jhomarolo) [reply package.json]: You still need a configuration file that is there in the todo-list-on-herbs repository.

I believe that we should put this file somewhere as a reference in the contribution rules file.

But if you want I can remove these files from the dev dependency.

File: sample.js (hunk 2 above)
Comment (jhomarolo) [reply sample.js]: Yes

File: package.json (hunk 1 above)
Comment (dalssoft) [reply package.json]: is there a way to automate it? 
like not letting people create a PR if the rules for lint are not been applied? 
like applying prettier automatically before a push?

Comment (dalssoft) [reply package.json]: btw, there should not be a dependency to todo-list-on-herbs

Comment (jhomarolo) [reply package.json]: I will check to see if there is something like that and return as a new issue.

For now I will remove the PR dependencies

**Review by dalssoft — APPROVED** [review]

**Pull request merged**